pom.xml

```diff
@@ -40,6 +40,12 @@
       <version>3.8.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
+   <!--   <type>bundle</type> -->
```

Comment:
Can you remove the commented out line here?

Reply:
Do you want the entire line removed?  Or just the comment?